### PR TITLE
Patch documentation: waves

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

File: symplyphysics/laws/waves/wavelength_of_standing_wave_in_string_with_fixed_ends.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-"""
+r"""
```
Comment (blackyblack) [inline symplyphysics/laws/waves/wavelength_of_standing_wave_in_string_with_fixed_ends.py:1]: Remove `r`

File: symplyphysics/laws/waves/frequency_of_electron_transition_in_hydrogen.py [hunk 2]
```diff
@@ -6,57 +6,47 @@
...
 
 law = Eq(
     transition_frequency,
-    rydberg_constant * ((1 / lower_principal_quantum_number**2) -
+    quantities.rydberg_frequency * ((1 / lower_principal_quantum_number**2) -
     (1 / higher_principal_quantum_number**2)))
-r"""
-:code:`nu = R_H * (1 / n_1^2 - 1 / n_2^2)`
+"""
+:code:`f = R_H * (1 / n_1^2 - 1 / n_2^2)`
```
[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/waves/frequency_of_electron_transition_in_hydrogen.py:47]: The powers didn't render correctly.

[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/waves/frequency_of_electron_transition_in_hydrogen.py:47]: For now, it gives `f = (-1 / n_2^2 + n_1^(-2)) * R_H`.

[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/waves/frequency_of_electron_transition_in_hydrogen.py:47]: I'll add an explanatory comment.

File: symplyphysics/laws/waves/displacement_in_interfering_waves.py [hunk 3]
```diff
@@ -136,13 +118,13 @@ def calculate_displacement(
     position_: Quantity,
     time_: Quantity,
 ) -> Quantity:
-    # pylint: disable=too-many-arguments, too-many-positional-arguments
+    # pylint: disable=too-many-arguments
```
[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/waves/displacement_in_interfering_waves.py:121]: Hm, my default `pylint` installed doesn't give any warning.

[the remark this answers is not in the document]
Comment (blackyblack) [reply symplyphysics/laws/waves/displacement_in_interfering_waves.py:121]: You can try to install most recent version.

[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/waves/displacement_in_interfering_waves.py:121]: Yes, now it gives the warning.

File: symplyphysics/laws/waves/average_power_of_sinusoidal_wave_on_stretched_string.py [hunk 4]
```diff
@@ -12,61 +12,36 @@
...
 """
-
-wave_phase_velocity = Symbol("wave_phase_velocity", units.velocity)
+:symbols:`linear_density` of the string.
 """
-Phase velocity of the wave.
 
-Symbol:
-    :code:`v`
+wave_phase_velocity = symbols.phase_speed
```
[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/waves/average_power_of_sinusoidal_wave_on_stretched_string.py:32]: IMO it's not obvious what kind of speed applies to waves inherently. It makes sense to notate it as `phase_speed`, so as not to confuse the people reading. I guess in literature people often shorten it to just `speed` and write `wave speed` but I think we should be clear in the terms we're using.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**